examples/sui-move/first_package/sources/example.move

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module first_package::example {
+    use sui::object::{Self, UID};
+    use sui::transfer;
+    use sui::tx_context::{Self, TxContext};
+
+    struct Sword has key, store {
+        id: UID,
+        magic: u64,
+        strength: u64,
+    }
+
+    struct Forge has key {
+        id: UID,
+        swords_created: u64,
+    }
+
+    /// Module initializer to be executed when this module is published
+    fun init(ctx: &mut TxContext) {
+        let admin = Forge {
+            id: object::new(ctx),
+            swords_created: 0,
+        };
+
+        // transfer the forge object to the module/package publisher
+        transfer::transfer(admin, tx_context::sender(ctx));
+    }
+
+    // === Accessors ===
+
+    public fun magic(self: &Sword): u64 {
+        self.magic
+    }
+
+    public fun strength(self: &Sword): u64 {
+        self.strength
+    }
+
+    public fun swords_created(self: &Forge): u64 {
+        self.swords_created
+    }
+
+    /// Constructor for creating swords
+    public entry fun new_sword(
+        forge: &mut Forge,
+        magic: u64,
+        strength: u64,
+        ctx: &mut TxContext,
+    ): Sword {
+        forge.swords_created = forge.swords_created + 1;
+        Sword {
+            id: object::new(ctx),
+            magic: magic,
+            strength: strength,
+        }
+    }
+
+    // === Tests ===
+    use sui::test_scenario as ts;
+
+    #[test]
+    public fun test_module_init() {
```

Comment:
Maybe create constants for the addresses to make the tests easier to follow?

Reply:
I actually stripped those constants out initially, because that was the style in most of the tests I'd seen so far, but you're right that it's probably clearer with them.  I'll add them here and in the other examples too.